adapters/InMobi/Public/Headers/GADMAdapterInMobi.h

```diff
@@ -0,0 +1,32 @@
+//
```

Comment:
This file need not be a Public header file. Instead this can be part of the bundle under `InMobi/InMobiAdapter/`. Please move this file into `InMobi/InMobiAdapter/'.